docs/API.md

```diff
@@ -55,6 +55,8 @@ If the request object is invalid JSON, an error with code `-32700` is reported.
 
 If no method is specified or an invalid method is specified, an error with code `-32601` is reported.
 
+If no `params` object is specified, or the `param` object for the specified method is invalid, an error with code `-32602` is reported. For more information on the validation error data format see [Validation error data]. If an optionnal `language` field containing a [Language tag] is specified within the `params` object, error messages will be translated to the target language,
```

Comment:
Please keep the lines by a maximum of 80 characters.

Some API methods do no have any params object, what happens to them?

"optionnal" -> "optional"

The language field is mandatory in the "get_test_results" method. That collides with the statement that it is optional above.

If the "language" field is added to all or some methods,  I think it must be specified by the method. I think we could start by adding it to the "start_domain_test", "get_host_by_name" and "get_data_from_parent_zone".

Reply:
Now, only the stated methods support the language field in param.